bin/coreos-install

```diff
@@ -36,6 +38,7 @@ USAGE="Usage: $0 [-V version] [-d /dev/device]
 Options:
     -d DEVICE   Install CoreOS to the given device.
     -V VERSION  Version to install (e.g. current) [default: ${VERSION_ID}]
+    -B BOARD    CoreOS board to use [default: ${BOARD}]
```

Comment:
(It won't let me comment under under the `-b` line.)  I'd suggest appending something like " (overrides BOARD)" to the `BASEURL` option's description so users are aware of that behavior.